parsers/wiki_parser.py

```diff
@@ -240,20 +210,27 @@ def links_for_files(files_list):
     files_link_mapping = {}
     futures = []
     empty_api_length = len(requests.get(url=API_PATH, params=get_params_for_files_api([])).url)
+    max_title_list_length = 50
+    file_title_prefix_length = len('%7CFile%3A')  # url encoded file title prefix '|File:'
```

Comment:
I don't remember how it should be handled in Python, but can't it be a constant defined before the function (maybe at the start of the file)? Now it's not obvious and there's a need to double check if the value does not change further in the function, does not seem like a constant (in Java it's handled like this: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names ).

Reply:
Well if it was Java or C++, sure, but Python doesn't have constant variable support. There are workarounds like having a class with only getter method for the property, but it's not fool proof.